bgpd/bgpd.c

```diff
@@ -7249,6 +7249,7 @@ void bgp_master_init(struct thread_master *master, const int buffer_size)
 	bm->rmap_update_timer = RMAP_DEFAULT_UPDATE_TIMER;
 	bm->v_update_delay = BGP_UPDATE_DELAY_DEF;
 	bm->v_establish_wait = BGP_UPDATE_DELAY_DEF;
+	bm->list_soft_reconfig_table = list_new();
```

Comment:
This sure seems like it should be the property of a `struct bgp *` instead of all of bgp right?  If I have a soft reconfig going on in `router bgp vrf blue 99` why should I have a list of stuff or be bothered by what is going on in `router bgp vrf green 329`?

Reply:
OK https://github.com/FRRouting/frr/pull/7457/commits/22ac83ef09fe3d0c29df21e113574bf6aca67171